hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/SimpleContainerDownloader.java

```diff
@@ -67,6 +71,8 @@ public SimpleContainerDownloader(ConfigurationSource conf,
     }
     securityConfig = new SecurityConfig(conf);
     this.certClient = certClient;
+    this.compression = conf.get(HDDS_CONTAINER_REPLICATION_COMPRESSION,
+        HDDS_CONTAINER_REPLICATION_COMPRESSION_DEFAULT);
```

Comment:
Please handle unknown compression by falling back to `NO_COMPRESSION`.

Reply:
finished